components/fallback/fallback.cpp

```diff
@@ -22,15 +24,15 @@ namespace Fallback
         if(fallback.empty())
             return 0;
         else
-            return boost::lexical_cast<float>(fallback);
+            return std::stof(fallback);
```

Comment:
Thank you for catching that, I would not have thought to test that. Outside the US that the , is used instead of the .